.circleci/config.yml

```diff
@@ -1488,9 +1488,9 @@ jobs:
             # Temporarily pin pillow to 6.2.1 as PILLOW_VERSION is replaced by
             # _version_ in 7.0.0. Long term fix should be making changes to
             # torchvision to be compatible with both < and >= v7.0.0.
-            pip install pillow==6.2.1
+            pip install pillow==6.2.1 --quite
```

Comment:
I think you misspelled "quiet"

Reply:
@zou3519 can you give a green check mark ?

Reply:
I don't know the consequences of this, a --quiet recently caused a timeout bug. I'll request review from the people who have more context about it.